### PR TITLE
Add denormalizer

### PR DESCRIPTION
**Pull request by sanpii**
Depends on https://github.com/pomm-project/pomm-symfony-bridge/pull/30

### CODE REVIEW

File: sources/lib/Resources/config/services/pomm.yml [hunk 1]
```diff
@@ -8,10 +8,16 @@ services:
         arguments: ['@pomm']
         tags:
             - { name: request.param_converter, converter: pomm.entity }
+
     pomm.serializer.normalizer:
         class: 'PommProject\SymfonyBridge\Serializer\Normalizer\FlexibleEntityNormalizer'
         tags:
-            - { name: serializer.normalizer }
+            - { name: serializer.normalizer, priority: 999 }
```
Comment (soyuka) [inline sources/lib/Resources/config/services/pomm.yml]: why `999`? I'm sure it can be a `<100` number that doesn't look "random" :stuck_out_tongue: no?

Comment (sanpii) [inline sources/lib/Resources/config/services/pomm.yml]: Yes, it’s totally random 😞

Comment (soyuka) [inline sources/lib/Resources/config/services/pomm.yml]: Hmm, but why? I mean what is the context where this normalizer will be used? Is it needed to declare the `serializer.normalizer` tag in this bundle (maybe leave this to the end user)?

Comment (sanpii) [inline sources/lib/Resources/config/services/pomm.yml]: If I recall well, it requires a priority to override the default normalizers provided by `symfony/serializer`.

Comment (soyuka) [inline sources/lib/Resources/config/services/pomm.yml]: Just use priority 0 or 1 then :p. Also it looks like priority should be between `-1` and `255`.

Comment (sanpii) [inline sources/lib/Resources/config/services/pomm.yml]: Ok, done. Thank you.

**Comment by sanpii** [comment]
Merged, thank you for your help 😃

**Pull request closed without merging**